dev-db/sqlcipher/sqlcipher-4.5.7.ebuild

```
@@ -0,0 +1,67 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic multilib-minimal
+
+DESCRIPTION="Full Database Encryption for SQLite"
+HOMEPAGE="https://www.zetetic.net/sqlcipher/"
```

Could you also add the Github page as 2ndary HOMEPAGE here?
```suggestion
HOMEPAGE="https://www.zetetic.net/sqlcipher https://github.com/sqlcipher/sqlcipher"
```